mmcif/api/DataCategory.py

```diff
@@ -446,6 +447,98 @@ def countValuesWhereOpConditions(self, conditionTupleList):
                 raise e
         return iCount
 
+    def selectIndicesWhereOpConditions(self, conditionTupleList):
+        """Select row indices subject to input condition list
+
+        Args:
+            conditionTupleList (list): (attributeName, op, value) where (op = 'eq', 'gt(int)', 'lt(int)', 'in', 'ne', 'not in')
+
+        Raises:
+            e: any failure
+
+        Returns:
+            List: Row indices satisfying input conditions
+        """
+        try:
+            rL = []
+
+            cTL = [(atName, op, value) for (atName, op, value) in conditionTupleList if atName in self._attributeNameList]
+            idxD = {atName: self._attributeNameList.index(atName) for (atName, op, value) in cTL}
+
+            for ii, row in enumerate(self.data):
+                ok = True
+                for (atName, op, v) in cTL:
+                    if op == "eq":
+                        ok = (v == row[idxD[atName]]) and ok
+                    elif op == "ne":
+                        ok = (v != row[idxD[atName]]) and ok
+                    elif op == "lt(int)":
+                        ok = (int(row[idxD[atName]]) < v) and ok
+                    elif op == "gt(int)":
+                        ok = (int(row[idxD[atName]]) > v) and ok
+                    elif op == "in":
+                        ok = (row[idxD[atName]] in v) and ok
+                    elif op == "not in":
+                        ok = (row[idxD[atName]] not in v) and ok
+                    if not ok:
+                        break
```

Comment:
I'm noticing now that you have this `break` here whereas the other methods don't (e.g., `countValuesWhereOpConditions`). Should we add this to those other ones too? Seems like a logical efficiency to me, versus wasting time iterating over things once it's `False`.